setup.cfg

```diff
@@ -56,6 +57,8 @@ include =
   odc*
 
 [options.package_data]
+odc.geo =
```

Comment:
And this to include any stubs?

1. we currently don't have stubs, should they be generated at packaging?
2. should this also go into MANIFEST.in for source distribution?

Reply:
Ok, I see, it's just an empty file marker.

Reply:
> should this also go into MANIFEST.in for source distribution?

I haven't needed to. The package data part seems to do the trick.